src/OpenTelemetry.Instrumentation.Http/HttpClientTraceInstrumentationOptions.cs

```diff
@@ -27,7 +27,7 @@ namespace OpenTelemetry.Instrumentation.Http;
 /// <summary>
 /// Options for HttpClient instrumentation.
 /// </summary>
-public class HttpClientInstrumentationOptions
+public class HttpClientTraceInstrumentationOptions
```

Comment:
Should we rename this to just `TraceInstrumentationOptions`? The namespace already has `OpenTelemetry.Instrumentation.Http`.

@alanwest What do you think?

Reply:
If you change it here to TraceInstrumentationOptions, we should made similar changes/recommendation to all other packages (here and in contrib).

Reply:
> That would cause ambiguous reference issues for users configuring options via Options API for e.g.

Ah good point!

Reply:
> Ah good point!

Agreed. Name seems good to me.

> @alanwest What do you think?

sorry... didn't see this. I used to get push notifications on my phone when @'ed, but I must of done something cause I didn't get one in this case... gonna try to fix this